src/monocle_apptrace/metamodel/spans/span_format.json

```diff
@@ -0,0 +1,102 @@
+{
+    "name": "span-name",
+    "context": {
+        "trace_id": "trace-id",
+        "span_id": "span-id",
+        "trace_state": "[]"
+    },
+    "kind": "SpanKind.CLIENT",
+    "parent_id": "parent-id or None (for root span)",
+    "start_time": "UTC timestamp",
+    "end_time": "UTC timestamp",
+    "status": {
+        "status_code": "OK or Error"
+    },
+    "attributes": {
+        "description": "List of AI component entities used in this span, eg Model, Inference hosting service. Needs to be one of the supported entity types.",
+        "Monocle.Entities.list": ["Entity-name-1", "Entity-name-2"],
+        "Monocle.Entity.<name>.type": "Monocle-Entity-Type",
```

Comment:
entity names can also have dot I think, eg gpt-3.5-turbo-0125, it could cause conflict with using dot as separator as well, eg Monocle.Entity. gpt-3.5-turbo-0125.type